libr/core/cmd_anal.c

```diff
@@ -1736,11 +1736,19 @@ static int anal_fcn_list_bb(RCore *core, const char *input, bool one) {
 	}
 	RAnalFunction *fcn = r_anal_get_fcn_in (core->anal, addr, 0);
 	if (!fcn) {
+		if (mode == 'j') {
+			j = pj_new ();
+			pj_a (j);
+			pj_end (j);
+			r_cons_printf ("%s\n", pj_string (j));
+			pj_free (j);
```

Comment:
make "j" local or set it to null after that, otherwise it can easily result into a doublefree or uaf